config/webpack.config.js

```diff
@@ -237,6 +237,7 @@ const pluginConfig = function(plugin) {
   configModules.unshift(path.resolve(__dirname, '..', 'node_modules'));
   // add plugin's node_modules to the reslver list
   configModules.push(path.resolve(pluginRoot, 'node_modules'));
+  configModules.push('node_modules/');
```

Comment:
Which values end up in `configModules`? Is `pluginRoot` perhaps wrong?

Reply:
`/usr/lib/node_modules/$package/node_modules` is not `pluginRoot`ed

Reply:
https://github.com/theforeman/foreman/blob/6c91c922305d5439ecae3574d7d202854af572dd/config/webpack.config.js#L73-L77
this is where the idea comes from (3.9 branch)

Reply:
(and https://webpack.js.org/configuration/resolve/#resolvemodules for reference how this works out with absolute paths generated by `path.resolve` and relative ones like the one I'm adding)

Reply:
Ah, so it's not considering the system wide installed NodeJS packages anymore, but it previously was. That helps my understanding.

Reply:
> Which values end up in configModules?
```
[
  '/home/mariaaga/git/foreman/node_modules',
  '/home/mariaaga/git/foreman_puppet/node_modules',
  'node_modules/'
]
```
which looks correct